tests/functional/adapter/dynamic_table_tests/test_dynamic_tables_changes.py

```diff
@@ -103,6 +95,9 @@ def setup(self, project, my_dynamic_table):
         # the tests touch these files, store their contents in memory
         initial_model = get_model_file(project, my_dynamic_table)
 
+        # verify the initial settings are correct in Snowflake
+        self.check_start_state(project, my_dynamic_table)
```

Comment:
This used to be commented at the beginning of every test. It was moved here and will run automatically. So it looks like the comment was removed from each test, but it as effectively just uncommented.